tox.ini

```diff
@@ -22,9 +22,10 @@ deps =
     sqlitedict==1.5.0
     url-matcher==0.2.0
     web-poet==0.7.0
+    Twisted<=22.10.0
```

Comment:
This only affects pinned environments, and in fact ideally we should have the lowest Twisted version fixed here. Where you need to add this change is in [testenv]deps above.

Reply:
Fixed, should I set lowest Twised version here?